src/assets/crisistextline/sass/partials/03-components/_user-form.scss

```diff
@@ -0,0 +1,22 @@
+.user-form {
+  .radio-input {
+    display: inline-block;
+    overflow: hidden;
+
+    input, label {
+        float: left;
+    }
+    input {
+        clear: both;
+    }
+}
+
+  div {
+    margin-top: 5px;
+    margin-bottom: 5px;    
+  }
+
+  label.top {
+    vertical-align: top;
+  }
```

Comment:
Any likelihood we'll want to replicate this vertically-stacked-checkboxes style for other forms that aren't explicitly `user-form`s?

Reply:
Maybe - I took some of the styles from the referral form where they were embedded among lots of other styles/classes. Maybe for now let's leave it until we have another place to use it and we can rename?